src/index.mako.html

```diff
@@ -464,7 +466,7 @@
                 </a>
               </p>
               <p>
-                <a ng-href="{{deviceSwitcherHref}}" translate>desktop_redirect</a>
+                <a href="{{deviceSwitcherHref}}" translate>desktop_redirect</a>
```

Comment:
The pulldown is not displayed until the translation is ready, so the ng-href is useless here.